webapp/src/monaco.tsx

```diff
@@ -1606,6 +1606,9 @@ export class Editor extends toolboxeditor.ToolboxEditor {
         monacoFlyout.style.display = 'block';
         monacoFlyout.className = 'monacoFlyout';
         monacoFlyout.style.transform = 'none';
+        monacoFlyout.addEventListener("wheel", e => {
```

Comment:
Don't we reuse this div? We should only register the event once and we should remove it when the flyout is hidden.